### ISSUE TITLE
User displayed as manager but not defined as such

### ISSUE THREAD

**Issue by amotenko** [Midburn/spark#758]
Camp id 2034 has no members defined as managers in `camp_members`.
User id 6438 is displayed as manager.

(Production)

**Comment by amotenko** [comment]
@ArtyomKa says it's a feature not a bug.
If no manager defined, main contact displayed as manager?

**Comment by ArtyomKa** [comment]
No. I said a that being main contact is enough to get manager permissions.

On Mar 13, 2018 11:12, "Alex" <notifications@github.com> wrote:

@ArtyomKa <https://github.com/artyomka> says it's a feature not a bug.
If no manager defined, main contact displayed as manager?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/Midburn/spark/issues/758#issuecomment-372596312>, or mute
the thread

**Comment by amotenko** [comment]
So the bigger issue here is that 
a. managers should be defined as such in the database
b. main contact shouldn't be manager by default, main contact is the person defined as manager

I'll open a new issue for improving the whole manager / main contact / other contacts thing
